homeassistant/components/alexa/smart_home.py

```diff
@@ -1839,11 +1839,17 @@ def temperature_from_object(hass, temp_obj, interval=False):
         climate.ATTR_OPERATION_MODE: ha_mode,
     }
 
+    response = directive.response()
     await hass.services.async_call(
         entity.domain, climate.SERVICE_SET_OPERATION_MODE, data,
         blocking=False, context=context)
+    response.add_context_property({
```

Comment:
Alternative is to change `blocking=False` to `blocking=True` and we get the real value.

Reply:
You know I thought I tried that once, and it didn't work. I didn't dig into it. In any case, I'd expect that to increase response latency, which might not be good for UX. There is https://developer.amazon.com/docs/device-apis/alexa-interface.html#deferredresponse, but I don't know how that would impact UX.

Reply:
the reason it doesn't always work is because it will only work with polled devices. It won't work with devices that push their state to HA, as that state change will be received too late for the service to be marked done. I think that for now I am fine with pretending it worked.